server.js

```diff
@@ -34,10 +37,41 @@ app.get('/sayHello', (request, response) => {
   // notice we can get proof of life. LOGS show IN TERMINAL
   // console.log(request.query);
   // console.log(name);
-  response.status(200).send(`Hello ${name}, from the Server`);
+  response.status(200).send(`Hello ${name}, from the Server`); //renders to browser
 
-});
+}); */
+
+/* //to hit this route:http://localhost:3002/throw-an-error
+
+app.get(’/throw-an-error’, (request, response)⇒ {
+
+  throw ‘You did something  really really bad!’;//for critical errors like sledgehammer
+
+})
+
+//no validation is drawback */
+
+//above-some useful things
+// ______
+
+//create route ‘/pets’ your will be ‘/weather’ that uses json to send a tailored response
 
+// hit this route: http://localhost:3002/pets? species=dog
+// hit this route: http://localhost:3002/weather? city_name=Seattle
+
+app.get('/weather', (request,response) => {
+
+  let city_name = request.query.city_name;
+
+  //gives proof of life in the TERMINAL
+
+  console.log(city_name);
+
+  response.send(weatherData.filter(city => city.city_name === city_name)[0].data.map(dataDay => new Forecast(dataDay)));  //chained!!! 
+
+  // output
+
+})
```

Comment:
You are on your way here!

- What is next is if the user did not search for one of the three cities we have info about, we should return an error (you are already using res.status from Express so we can use that)
- Then we just need to eventually send the data array to the client who will be asking for it 👍 
- We will want to work in a 500 server error as well